hugo/package.json

```diff
@@ -38,19 +39,19 @@
     "clsx": "^1.2.1",
     "d3": "^7.8.5",
     "hugo-extended": "~0.89.4",
-    "langium": "^1.2.0",
-    "langium-arithmetics-dsl": "~1.2.1",
-    "langium-domainmodel-dsl": "~1.2.0",
-    "langium-minilogo": "^1.2.0",
-    "langium-sql": "^0.2.3",
-    "langium-statemachine-dsl": "~1.2.0",
+    "langium": "~3.0.0",
```

Comment:
We should uniformly define dependencies that has overlaps in the dependency tree, either relaxed `^` or more strict `~`

Reply:
What do you mean? How should my action look like?

Reply:
Ah I think I see what you meant ^^...

Reply:
Done